Formula/wine.rb

```diff
@@ -134,6 +489,16 @@ def caveats
     s
   end
 
+  def post_install
+    # For fontconfig
+    ohai "Regenerating font cache, this may take a while"
+    system "#{libexec}/bin/fc-cache", "-frv"
+
+    # For net-snmp
+    (var/"db/net-snmp").mkpath
+    (var/"log").mkpath
+  end
+
   test do
     system "#{bin}/wine", "--version"
```

Comment:
Ideally I think we'd just use a `resource` for some downloaded command-line `.exe` that we actually run.